common.blocks/popup/popup.tmpl-specs/10-id.bemjson.js

```diff
@@ -0,0 +1,4 @@
+({
+    block : 'popup',
+    id : 1
```

Comment:
Please note: `id`s are `String`, according to [our documentation][1] (_int53 is too small for current web apps_)

[1]: https://en.bem.info/libs/bem-components/v2.0.0/desktop/button/docs/#buttonid